q2_fragment_insertion/_insertion.py

```diff
@@ -302,3 +303,43 @@ def classify_otus_experimental(
              "i.e. are results from the same 'sepp' run."))
 
     return pd_taxonomy.set_index('Feature ID')
+
+
+def filter_features(table: biom.Table,
+                    tree: NewickFormat) -> (biom.Table, biom.Table):
+
+    # load the insertion tree
+    tree = skbio.TreeNode.read(str(tree))
+    # collect all tips=inserted fragments+reference taxa names
+    fragments_tree = {
+        str(tip.name)
```

Comment:
Why is casting to `str` needed?

Reply:
see above; short: I am afraid of numbers parsed as `int`